site/en/blog/deprecating-web-sql/index.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Deprecating Web SQL
```

Comment:
```suggestion
title: Deprecating and removing Web SQL
```

Reply:
If this article is going to be updated with the future status, then it's title needs to be broader.